src/NodaTime.Web/Startup.cs

```diff
@@ -80,6 +84,43 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                 ContentTypeProvider = new FileExtensionContentTypeProvider
                 {
                     Mappings = { [".nzd"] = "application/octet-stream" }
+                },
+                OnPrepareResponse = (context) => {
+                    // Set the Cache-Control header for static content.
+                    var headers = new ResponseHeaders(context.Context.Response.Headers);
+
+                    // All static content is cacheable by intermediates.
+                    headers.CacheControl = new CacheControlHeaderValue()
```

Comment:
Pass. I'm far from expert on this. However, this is all just about serving static content, yes? Should we do something about the Markdown we serve as well?